### ISSUE TITLE
Add disability income tab in UI to trigger logic for SSI/SSDI/etc.

### ISSUE THREAD

**Issue by nmrodelo** [PolicyEngine/policyengine-app#452]
Per a discussion I had with someone testing the app, it would be valuable to have disability income for SSI/SSDI included in the tabs for calculating household income. This would allow users to more easily verify those benefits are being computed in the calculation and affecting other variables (SSDI, for instance, has huge impacts on SNAP), and perhaps we could even use this as a way to benchmark how far we are off from the reality of what people are receiving and flag issues with our calcs.

**Comment by MaxGhenis** [comment]
Thanks Nick, I moved this to the app repo. I think this would be better handled in a more general way, e.g. the tile drill-down layout we have for parameters instead of requiring searching (at least as a start) but good to know this use case.

**Comment by nikhilwoodruff** [comment]
Yeah- the current setup of the app only lets people enter in non-formula variables. So to fix this we'd need to reassess that

**Comment by MaxGhenis** [comment]
Thank you for filing this issue. Closing as part of the `policyengine-app` (v1) wind-down: policyengine.org is now served by [`policyengine-app-v2`](https://github.com/PolicyEngine/policyengine-app-v2), and this repository is being archived, so this issue is frozen against a codebase that no longer deploys anywhere.

If this is still relevant on the current site, please re-file it against [`policyengine-app-v2`](https://github.com/PolicyEngine/policyengine-app-v2) — we'd welcome it there. Apologies that this sat open through the migration.
